db/migrate/20220309132719_copy_post_uploads_to_upload_references.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CopyPostUploadsToUploadReferences < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+      INSERT INTO upload_references(upload_id, target_type, target_id, created_at, updated_at)
+      SELECT upload_id, 'Post', post_id, NOW(), NOW()
+      FROM post_uploads
+      ON CONFLICT DO NOTHING
```

Comment:
I am not sure if this will work, I am 95% sure that currently an upload can be linked to multiple posts, which has been an issue with secure media things. I don't think we can have this or the unique index on type/id/upload_id